pkg/test/framework.go

```diff
@@ -105,6 +108,9 @@ func (opts *frameworkOpts) addToFlagSet(flagset *flag.FlagSet) {
 	flagset.BoolVar(&opts.singleNamespaceMode, SingleNamespaceFlag, false, "enable single namespace mode")
 	flagset.StringVar(&opts.localOperatorArgs, LocalOperatorArgs, "",
 		"flags that the operator needs (while using --up-local). example: \"--flag1 value1 --flag2=value2\"")
+	flagset.BoolVar(&opts.skipCleanupOnError, SkipCleanupOnErrorFlag, false,
+		"The clean up function responsible to remove all artifacts will be skipped if "+
```

Comment:
Based on @jmccormick2001 , wdyt about:
```suggestion
		"If set as true, the cleanup function responsible to remove all artefacts will be skipped if an error is faced."+
```

Reply:
Shows solved.